libsplinter/src/admin/store/event.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2018-2020 Cargill Incorporated
```

Comment:
The copyright in this PR should be updated to 2021

Reply:
Updated